src/Pages/History-page/History.jsx

```diff
@@ -0,0 +1,18 @@
+import "../Home-Page/Home.css";
+import "../Home-Page/Home-Responsive.css";
+import { Nav, NavForTab, Sidebar } from "../../components/Index";
+import { HistoryCard } from "./Components/HistoryCard.jsx";
+import { Header } from "./Components/Header";
+export function History() {
+  return (
+    <div className="page--Container">
+      <Nav hideComponent={false} />
+      <NavForTab />
+      <Sidebar />
```

Comment:
Suggestion : you separate out Layout and components for clean structure for file management

Reply:
Updated ThankYou